utils/formats/sarifutils/sarifutils.go

```diff
@@ -417,6 +476,13 @@ func SetResultMsgMarkdown(markdown string, result *sarif.Result) {
 	result.Message.Markdown = &markdown
 }
 
+func GetResultMsgMarkdown(result *sarif.Result) string {
+	if result == nil || result.Message.Markdown != nil {
```

Comment:
this can lead to nil pointer dereference.
if result == nil you enter the condition and will get nil pointer in *result
did you mean if result != nil ?

Reply:
fixed